app/src/main/java/com/example/haushaltsbuch/data/model/DevData.kt

```diff
@@ -20,7 +20,7 @@ val fakerConfig = FakerConfig.builder().create { locale = "de" }
 val faker = Faker(fakerConfig)
 
 class FakeData(){
-
+    // Data
     var people: MutableList<Person>? = null
```

Comment:
Hier könnte man auch bspw. folgendes machen:
`val locations = List<Location>(locationCount, ::generateLocation)`

damit würden wir zum einen Code sparen, brauchen `null` nicht und können `List` statt `MutableList` verwenden.

Reply:
der gedanke ist, dass man die Funktionen auch außerhalb aufrufen kann

Reply:
Kann man so doch auch, nur statt `null` schreiben wir direkt die Werte in die Liste.